components/gitpod-cli/cmd/timeout-extend.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"context"
+	"time"
+
+	gitpod "github.com/gitpod-io/gitpod/gitpod-cli/pkg/gitpod"
+	"github.com/spf13/cobra"
+)
+
+// timeoutExtendCmd represents the timeoutExtendCmd command
+var timeoutExtendCmd = &cobra.Command{
+	Use:   "timeout-extend",
+	Short: "Boost the timeout of this workspace",
+	Args:  cobra.NoArgs,
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Please have a look how `snapshot` gp cli command is implemented to learn how to connect to Gitpod Server.

Reply:
Actually it looks good, besides what @iQQBot mentioned.